api/v1alpha3/vspherecluster_types.go

```diff
@@ -70,6 +70,10 @@ type VSphereClusterStatus struct {
 // +kubebuilder:resource:path=vsphereclusters,scope=Namespaced,categories=cluster-api
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this VSphereCluster belongs"
```

Comment:
not sure this is useful, most times the vspherecluster and cluster name are matching